webapp/src/translations/languages/en.json

```diff
@@ -651,7 +651,9 @@
         "addReceiveAddress": "Add a receive address",
         "address": "Address",
         "enterAnAddress": "Enter an address in your account",
-        "invalidAddress": "Invalid address. Please check this belongs to you, the current address has been used."
+        "existingAddressOthers": "Invalid address. Please check this belongs to you, the current address has been used.",
```

Comment:
I think we can remove "the current address has been used". It doesn't mean anything in this error message.